tools/api-tests/src/api-test/manuscript.test.ts

```diff
@@ -2,79 +2,197 @@ import { test, TestContext } from "ava-ts";
+        expertises: [],
+      },
+    ],
+    suggestedReviewers: [
+      {
+        name: "Some Other",
+        email: "guy@example.com",
+      },
+      {
+        name: "",
+        email: "",
       },
-    };
-
-    // TODO: Test updating other fields in the form - really exercise those database relations!
-    const savedManuscript = await submission.Mutation.updateSubmission(context, {data: manuscriptAuthorDelta});
-
-    const afterUpdating = await dashboard.Query.manuscripts(context);
-
-    // Check that the user has a new manuscript
-    t.truthy(
-      initialManuscripts.manuscripts.length < afterUpdating.manuscripts.length,
-      "there are more manuscripts after a new one is created",
-    );
-
-    // Check that the new manuscript exists in the users new manuscripts
-    t.truthy(
-      afterUpdating.manuscripts.filter(
-        (manuscript: Manuscript) => manuscript.id === savedManuscript.updateSubmission.id,
-      ).length === 1,
-      "the new manuscript exists in the updated list of manuscripts",
-    );
-
-    // Check that the new manuscript does not exist in the user's old manuscript list
-    t.truthy(
-      initialManuscripts.manuscripts.filter(
-        (manuscript: Manuscript) => manuscript.id === savedManuscript.updateSubmission.id,
-      ).length === 0,
-      "the new manuscript does not exist in the original list",
-    );
-
-    // TODO: Work out exactly what should be here
-    const expectedUpdatedManuscript: Manuscript | ManuscriptInput = {
-      ...newManuscript.createManuscript,
-      ...manuscriptAuthorDelta as Manuscript,
-      suggestedReviewers: [],
-    }
-
-    // Check that the stored manuscript matches the original
-    // NOTE: The server adds __typename properties that aren't defined in our schema
-    // and aren't used anywhere, so we strip them out to check that the stuff we depend on
-    // is right
-    t.deepEqual(
-      stripTypeNameFromJson(savedManuscript.updateSubmission),
-      stripTypeNameFromJson(expectedUpdatedManuscript),
-      "the manuscript that is saved is the same as the original manuscript",
-    );
-  },
-);
+    ],
+  };
+
+  const savedManuscript = await submission.Mutation.updateSubmission(context, {
+    data: manuscriptEditorsDelta,
+  });
+
+  const expectedUpdatedManuscript: Manuscript = {
+    ...newManuscript.createManuscript,
+    ...(manuscriptAuthorDelta as Manuscript),
+    ...hydratedTeams,
+  };
+
+  const fetchedManuscript = await submission.Query.getSubmission(context, {  id: newManuscript.createManuscript.id });
+
+  // NOTE: The server adds __typename properties for use internally, we don't need to assert on them so I've
+  // removed them
+  t.deepEqual(
+    stripTypeNameFromJson(expectedUpdatedManuscript),
+    stripTypeNameFromJson(savedManuscript.updateSubmission),
+    "the saved manuscript contains the correct teams data",
+  );
+
+  t.deepEqual(
+    // Huh?: why do we get suggestions back with this one and not the other request?
```

Comment:
Has this been raised as an investigation issue if its not intended functionality?

Reply:
Not quite yet, will add now :+1: